frontend/src/views/OmakasePlayer/components/OmakaseSegmentation/OmakaseSegmentationHeader.tsx

```diff
@@ -22,10 +28,12 @@ const OmakaseSegmentationHeader = ({
   source,
   onSegementationClickCallback,
   flows,
-  flowSegments,
   markerOffset,
   omakasePlayer,
 }: OmakaseSegmentationHeaderProps) => {
+  const [editTimeranges, setEditTimeranges] = useState();
+  // @ts-ignore
+  const { setOmakaseModalVisible } = useOmakaseStore((state) => state);
```

Comment:
As per my comment above on the zustand store i just feel a piece of local state here that we pass into our component as props might be nicer?

Reply:
good point - ive moved this to local state